src/js/components/aboutTheData/AgencyDetailsPage.jsx

```diff
@@ -0,0 +1,126 @@
+/**
+ * AgencyDetailsPage.jsx
+ */
+
+import React, { useEffect, useState } from 'react';
+import { useParams, Link } from 'react-router-dom';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { LoadingMessage, ErrorMessage } from 'data-transparency-ui';
+
+import { agencyPageMetaTags } from 'helpers/metaTagHelper';
+import { mockFetchAgencyOverview } from 'helpers/agencyV2Helper';
+
+import MetaTags from 'components/sharedComponents/metaTags/MetaTags';
+import Header from 'containers/shared/HeaderContainer';
+import Footer from 'containers/Footer';
+import StickyHeader from 'components/sharedComponents/stickyHeader/StickyHeader';
+import Note from 'components/sharedComponents/Note';
+import AgencyDetailsContainer from 'containers/aboutTheData/AgencyDetailsContainer';
+import { modalTitles, modalClassNames } from 'dataMapping/aboutTheData/modals';
+import BaseAgencyOverview from 'models/v2/agencyV2/BaseAgencyOverview';
+import ExternalLink from 'components/sharedComponents/ExternalLink';
+import AboutTheDataModal from './AboutTheDataModal';
+
+require('pages/aboutTheData/agencyDetailsPage.scss');
+
+const message = 'All numeric figures in this table are calculated based on the set of TAS owned by each agency, as opposed to the set of TAS that the agency directly reported to USAspending.gov. In the vast majority of cases, these are exactly the same (upwards of 95% of TAS—with these TAS representing over 99% of spending—are submitted and owned by the same agency). This display decision is consistent with our practice throughout the website of grouping TAS by the owning agency rather than the reporting agency. While reporting agencies are not identified in this table, they are available in the Custom Account Download in the reporting_agency_name field.';
+
+const AgencyDetailsPage = () => {
+    const { agencyCode } = useParams();
+    const [loading, setLoading] = useState(true);
+    const [error, setError] = useState(false);
+    const [errorMessage, setErrorMessage] = useState('');
+    const initialAgency = Object.create(BaseAgencyOverview);
```

Comment:
can we put this outside the component definition 🤔

Reply:
We might be able to get rid of it actually.. I was experimenting with putting mock data in the model but now the loading state should prevent accessing `agencyOverview` before it gets populated.